client/src/services/releases.ts

```diff
@@ -69,19 +68,19 @@ export async function fetchRelease (
   // We skip runtime type checks for simplicity (here we cast from `any` to `GithubRelease`)
   const release: GithubRelease = await response.json()
   const flixRelease: FlixRelease = {
-    url: _.get('url', release),
-    id: _.get('id', release),
-    name: _.get('name', release),
-    version: tagToVersion(_.get('tag_name', release)),
-    downloadUrl: _.get('assets.0.browser_download_url', release),
+    url: _.get(release, 'url'),
```

Comment:
Did the argument order change?

Reply:
Yes, as lodash get type definition, the function works well (I have tested it)

Reply:
it's somewhat strange I agree, but the type definition differs from what was written here

Reply:
Surprising!